### ISSUE TITLE
Not compatible with WebStorm 2024.3

### ISSUE THREAD

**Issue by mzeitz1902** [asnowwolf/angular-switch-extension#2]
The plugin is no longer working, could this be fixed?

**Comment by felhag** [comment]
Same applies for latest Intellij version :(

**Comment by mzeitz1902** [comment]
> Same applies for latest Intellij version :(

Yeah, I think this extension is dead, sadly.

**Comment by felhag** [comment]
Yeah that is real shame, apparently I really integrated this plugin into my workflow haha. Did you find any alternatives? I tried [this one](https://plugins.jetbrains.com/plugin/19692-angular-file-switcher) but I prefer navigating to the files directly.

**Comment by mzeitz1902** [comment]
I'm using the native shortcut Ctrl+Alt+Home now. Works well enough.
![image](https://github.com/user-attachments/assets/14df27d6-4537-4fe0-b5fa-467a2297c07d)

https://www.jetbrains.com/guide/javascript/tips/jump-between-files-angular/

**Comment by felhag** [comment]
Seems like it just needed some version bumps. Not sure how @asnowwolf used to build this, but I created a fork and added the standard Intellij plugin template. A build which is compatible with the latest Intellij version can be downloaded [here](https://github.com/felhag/angular-switch-extension/releases/download/v0.6.1/angular-switch-extension-0.6.1.zip).

@asnowwolf Are you planning to publish an updated version to the Intellij plugin marketplace?

**Comment by korniychuk** [comment]
Thank you @felhag  🤝 . Your fork works for me. WebStorm 2024.3.4.
